packages/config/config/devices/0x0063/templates/config_template.json

```
@@ -0,0 +1,468 @@
+			{
+				"label": "Default",
+				"value": 255
+			}
+		]
+	},
+	"switch_level": {
+		"label": "Switch Level",
+		"description": "Set power level for plain switch functionality",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 99,
+		"defaultValue": 0
+	},
+	"dim_rate": {
+		"label": "Dim Rate",
+		"description": "Adjust the speed at which the ramps to a specific value other than 0 and FF",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Dim Quickly",
+				"value": 0
+			},
+			{
+				"label": "Dim Slowly",
+				"value": 1
+			}
+		]
+	},
+	"timeout_duration": {
+		"label": "Timeout Duration",
+		"description": "Amount of time the light stays on after motion stops",
+		"valueSize": 1,
+		"defaultValue": 5,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Test (5s)",
+				"value": 0
+			},
+			{
+				"label": "1 minute",
+				"value": 1
+			},
+			{
+				"label": "5 minutes",
+				"value": 5
+			},
+			{
+				"label": "15 minutes",
+				"value": 15
+			},
+			{
+				"label": "30 minutes",
+				"value": 30
+			},
+			{
+				"label": "Disable timeout",
+				"value": 255
+			}
+		]
+	},
+	"pir_sensitivity": {
+		"label": "PIR Sensitivity",
+		"valueSize": 1,
+		"defaultValue": 3,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Low",
+				"value": 1
+			},
+			{
+				"label": "Medium",
+				"value": 2
+			},
+			{
+				"label": "High",
+				"value": 3
+			}
+		]
+	},
+	"motion_sensitivity": {
+		"label": "Motion Sensor Sensitivity",
+		"valueSize": 1,
+		"defaultValue": 2,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "High",
+				"value": 1
+			},
+			{
+				"label": "Medium",
+				"value": 2
+			},
+			{
+				"label": "Low",
+				"value": 3
+			}
+		]
+	},
+	"operation_mode": {
+		"label": "Operation Mode",
+		"description": "Sets how the light responds to motion",
+		"valueSize": 1,
+		"defaultValue": 3,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Manual",
+				"value": 1
+			},
+			{
+				"label": "Vacancy",
+				"value": 2
+			},
+			{
+				"label": "Occupancy",
+				"value": 3
+			}
+		]
+	},
+	"light_sensor": {
+		"label": "Enable/Disable Light Sensing",
+		"valueSize": 1,
+		"defaultValue": 1,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Enabled",
+				"value": 1
+			}
+		]
+	},
+	"motion_sensor": {
+		"label": "Enable/Disable Motion Sensing",
+		"valueSize": 1,
+		"defaultValue": 1,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Enabled",
+				"value": 1
+			}
+		]
+	},
+	"ignore_start_level": {
+		"label": "Ignore Start-Level (Receiving)",
+		"description": "This dimmer will start dimming from its current level.",
```

I think this would be clearer.
```suggestion
		"label": "Start Dimming From Current Level",
		"description": "The start level in Z-Wave commands will be ignored, if enabled",
```